gandi/cli/modules/paas.py

```diff
@@ -29,6 +32,67 @@ def type_list(cls, options=None):
         return cls.safe_call('paas.type.list', options)
 
     @classmethod
+    def attach(cls, name, vhost, remote_name):
+        """Attach an instance's vhost to a remote from the local repository."""
+        remote_url = cls.git_remote(name, vhost)
+
+        ret = cls.execute('git remote add %s %s' % (remote_name, remote_url,))
```

Comment:
I'm not, but I figured I'd let this responsibility to git itself, at least for the moment.
If I were to change my mind about this, it'd mean handle :

- current directory is not a git repository
- remote already exists

Which is fine by me, what do you think ?